src/pages/Category.vue

```diff
@@ -61,7 +61,8 @@ export default {
       category: {},
       filters: { // filters should be set by category, and should be synchronized with magento
         color: ['red', 'blue'],
```

Comment:
Sizes and colors must be mapped to some database-related values. For example "color" in DB is: 169, 180 etc. The same with size.

Can you provide data model for that? separate key / value or label / caption - you name it :) - pairs would be great @patrykpiston

Reply:
I'll add a mechanism to fill out these dictionaries from database

Reply:
This is key to merge this into master :-)

Reply:
Sure, once I get some time today / tomorrow I'll get back to it and come up with something :) @pkarw